css/selectors/host.json

```diff
@@ -16,16 +16,40 @@
               "version_added": true
             },
             "edge": {
-              "version_added": false
+              "version_added": false,
+              "notes": "<a href='https://developer.microsoft.com/microsoft-edge/platform/status/shadowdom/'>Under consideration</a>"
             },
             "edge_mobile": {
-              "version_added": false
+              "version_added": false,
+              "notes": "<a href='https://developer.microsoft.com/microsoft-edge/platform/status/shadowdom/'>Under consideration</a>"
             },
             "firefox": {
-              "version_added": false
+              "version_added": "61",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "dom.webcomponents.shadowdom.enabled",
+                  "value_to_set": "true"
+                }
+              ],
+              "notes": [
+                "See <a href='https://bugzil.la/1205323'>bug 1205323</a>",
```

Comment:
Please do not add these bugs as notes. This meta information is nice to have in the commit or the pull request comment, but is not necessary in the data or much useful for MDN readers. It rather makes you curious if there are known bugs with the implementation, which there aren't.

Reply:
Can I at least keep the link to the [Shadow DOM meta bug](https://bugzil.la/1205323), like the rest of the Shadow DOM data in this repository?

Reply:
I might not have seen that for the other Shadow DOM stuff. If you think it would be more consistent, that's alright then.

Reply:
It can be seen in the [`ShadowRoot` browser compatibility](https://developer.mozilla.org/docs/Web/API/ShadowRoot#Browser_compatibility) section.

Reply:
Oh yes, that bug is still open, so it makes sense in this case. Sorry I thought both bugs were closed already. This is alright then.